database/migrations/file_upload_webhooks001.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * File: file_upload_webhooks.ts
+ * Created Date: Feb 10, 2022
+ * Copyright (c) 2022 Zeytech Inc. (https://zeytech.com)
+ * Author: Clint Pamperin (https://github.com/cpamp)
+ * -----
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class FileUploadWebhooks extends BaseSchema {
+  protected tableName = 'file_upload_webhooks'
+
+  public async up() {
+    this.schema.table(this.tableName, (table) => {
+      table
+        .enu('webhook_type', ['success', 'failure'], {
+          useNative: true,
+          enumName: 'enu_fua_webhook_type',
+        })
+        .defaultTo('success')
+    })
+  }
+
+  public async down() {
+    this.schema.table(this.tableName, (table) => {
```

Comment:
I think you still have to manually delete the enu_fua_webhook_type type in the down migrations.
`DROP TYPE enu_fua_webhook_type` or something like that.

Reply:
Looks like the only way we can do this is with `raw` but not all dbs support type. If I try to drop the type the migration fails for Sqlite3

Reply:
Oh good call.  Probably not the worst thing in the world if we make this enum non-native.  I think this migration will fail in postgres if you migrate up, down, then up again because the type will already exist.

Reply:
Updated. Ran into the same issue with storage type but left it for now.